incubator/hydra/templates/hydra_deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      name: {{ template "fullname" . }}
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      volumes:
+        - name: hydra-data
+        {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ template "fullname" . }}
+        {{- else }}
+          emptyDir: {}
+        {{- end }}
+        - name: hydra-secret
+          secret:
+            secretName: {{ template "fullname" . }}-secret
+      containers:
+        - name: hydra
+          image: {{ .Values.image }}:{{ .Values.imageTag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["hydra", "host", "--dangerous-auto-logon"]
+          volumeMounts:
+            - name: hydra-data
+              mountPath: {{ .Values.mountPath }}
+          ports:
+            - name: service
+              containerPort: 4444
+            - name: service2
+              containerPort: 4445
+          env:
+            - name: SYSTEM_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "fullname" . }}-secret
+                  key: system.secret
+            - name: DATABASE_URL
+              value: postgres://{{ .Values.postgresql.postgresUser }}:{{ .Values.postgresql.postgresPassword }}@{{ template "postgresql.fullname" . }}:5432/{{ .Values.postgresql.postgresDatabase }}?sslmode=disable
+            - name: HTTPS_ALLOW_TERMINATION_FROM
+              value: 0.0.0.0/0
```

Comment:
Are you referring to the `HTTPS_ALLOW_TERMINATION_FROM` environment?

If you're referring to the postgres URL,  this was one thing I was pretty unsure about.  the `requirements.yaml` file seems to imply that a new postgres server will spin up for hydra regardless if there's one there already.  Not sure how that works.